demo/index.html

```diff
@@ -16,10 +16,10 @@
 
   <title>quick-element</title>
 
-  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+  <script src="../bower_components/webcomponentsjs/webcomponents-lite.js"></script>
```

Comment:
Assuming that the demo will be opened by people cloning `quick-element` locally and using `bower install` to get the dependencies, I'd expect a working demo when I open up the `demo/index.html` file in my browser without having other configurations made on my system.